public/pages/CreateMonitor/components/MonitorExpressions/MonitorExpressions.tsx

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
I couldn't tell what changed in this file since it was renamed as well

Reply:
Sorry, you would have to manually compare it to: https://github.com/opensearch-project/alerting-dashboards-plugin/blob/feature/anywhere/public/pages/CreateMonitor/components/MonitorExpressions/MonitorExpressions.js